src/test/java/setCollectionTest.java

```diff
@@ -0,0 +1,40 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.*;
+public class setCollectionTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+    @Test
+    void 요구사항1(){
+        assertThat(numbers.size()).isEqualTo(3);
+    }
+    @ParameterizedTest
```

Comment:
```suggestion
    }
    
    @ParameterizedTest
```
    
<img width="731" alt="스크린샷 2023-04-05 오후 8 28 21" src="https://user-images.githubusercontent.com/12059592/230067490-d8fe52c4-09c7-456a-b9fc-386af9f881c1.png">


----

개행처리 또한 코드컨벤션일수있어요~!